tasks/main.yml

```diff
@@ -7,6 +7,11 @@
 - name: Include OS variables
   include_vars: "{{ ansible_os_family }}.yml"
 
+- name: Set nomad_servers to empty list rather than the default macro if we are using consul
```

Comment:
Another unrelated oddity, is that because I'm running it with -c local, it makes the inventory_hostname 127.0.0.1, so the inventory_hostname_short is 127, lol, so just going to override in my playbook nomad_node_name: "{{ansible_hostname}}"